app.js

```diff
@@ -0,0 +1,16 @@
+var express = require("express"),
+	app 	= express();
+
+	app.set("view engine", "ejs");
+
+	app.get("/", function(req,res){
+		res.redirect("index");
+	})
+
+	app.get("/index", function(req, res){
+		res.render("index");
+	});
+
+	app.listen(3000, function () {
```

Comment:
Since we'll be merging to master/release branch. We should use 80 port here, which is default for HTTP. Also, we should move this into a constant variable and also change it from the console.log.

For ex.

var PORT = 80;
app.listen(PORT, function(){ ...